render/render.js

```diff
@@ -428,7 +440,7 @@ module.exports = function($window) {
 						}
 					}
 					removeNodeFromDOM(vnode.dom)
-					if (context != null && vnode.domSize == null && !hasIntegrationMethods(vnode.attrs) && typeof vnode.tag === "string") { //TODO test custom elements
+					if (context != null && vnode.reuse) { //TODO test custom elements
```

Comment:
I think that comment can go now. Custom elements are not recycled anymore (at least they shouldn't be if the code is correct). If a user opts in, it's not our problem :-)